### PR TITLE
v2raya: depends on v2raya-core

### PR DESCRIPTION
**Pull request by rocka**
xray provides `v2raya-core` since https://github.com/archlinuxcn/repo/commit/a45645ba7ea57e2d3b0753b5b41667f2e97df87b

### CODE REVIEW

File: archlinuxcn/v2raya/PKGBUILD [hunk 1]
```diff
@@ -7,8 +7,8 @@ pkgrel=1
 pkgdesc="A web GUI client of Project V which supports VMess, VLESS, SS, SSR, Trojan and Pingtunnel protocols"
 arch=('x86_64')
 url="https://github.com/v2rayA/v2rayA"
-license=('AGPL3')
-depends=('glibc')
+license=('AGPL-3.0')
```
Comment (zjuyk) [inline archlinuxcn/v2raya/PKGBUILD:10]: Thanks for the correction!

File: archlinuxcn/v2raya-core/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+pkgver=1
+pkgrel=1
+url='https://github.com/v2fly/v2ray-core'
+license=('MIT')
+depends=('v2ray')
+arch=('any')
+
+package() {
+    # nothing to package
+}
```
Comment (lilydjwg) [inline archlinuxcn/v2raya-core/PKGBUILD:14]: This is bash...

Comment (rocka) [reply archlinuxcn/v2raya-core/PKGBUILD:14]: didn't know this before 🥲

should have wrote `:;` instead?

Comment (lilydjwg) [reply archlinuxcn/v2raya-core/PKGBUILD:14]: Or `true`. `;` is not needed.

**Pull request merged**